modules/cnf-disable-chronyd.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+[id="cnf-disable-chronyd_{context}"]
+= Disabling chrony time service
+
+You can disable the chrony time service (`chronyd`) for nodes with a specific role using a `MachineConfig` custom resource (CR).
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
```

Comment:
It's just called the "OpenShift CLI". Usually we like to write it as "OpenShift CLI (`oc`)".